src/content/manifest/okteto-variables.mdx

```diff
@@ -1,15 +1,36 @@
 ---
 title: Okteto Variables
-description: Okteto Variables allows you to save application configuration in Okteto, and automatically inject them during deployment time
-sidebar_label: Okteto variables
+description: Okteto Variables allows you to save application configuration in Okteto and automatically inject them during deployment time
+sidebar_label: Variables
 id: okteto-variables
 ---
 
 import Image from "@theme/Image";
 
 Application configuration should be passed at deployment time, not hardcoded in your code.
 
-Okteto Variables allows you to save application configuration in Okteto, and automatically inject it during deployment time.
+Okteto variables help you save application configuration in Okteto and automatically inject it during deployment time.
+
+There are a few different ways to leverage variables. In short:
+
+- You can define variables manually in the manifest or in the Okteto UI
+- There are some default variables built in to the Okteto platform
+- All variables used in the Okteto manifest are automatically expanded
+- Variables referenced in the `deploy` and `destroy` sections are automatically injected into those environments
+
+We will cover each of these behaviors in more detail below.
+
+## User-defined variables
+
+Okteto supports manual definition of variables, offering significant flexibility in configuration. These user-defined variables are separate from Okteto's default, built-in variables. We recommend creating variables in the following order of preference:
+
+- Local variables
+- Deploy time variables
+- Catalog variables
+- User variables (in settings)
+- Admin variables
+
+> Note: Variables defined using methods lower in this list (e.g. admin variables) will override mentions of the same variable defined by methods higher in this list (e.g. deploy time variables)
```

Comment:
@pchico83 I added this because I assume this is the case. How should I correct this?

Reply:
Preference is in reverse order. For example, a deploy time variable overrides an admin variable

Reply:
Thank you! Will push change in next commit 👍